src/payment/data/selectors.js

```diff
@@ -1,27 +1,27 @@
 import { createSelector } from 'reselect';
-import {
-  localeSelector,
-  getCountryList,
-} from '@edx/frontend-i18n';
+import { localeSelector, getCountryList } from '@edx/frontend-i18n';
```

Comment:
I mildly refactored the contents of this file to be a bit more canonical.  Still needs a little work.